### PR TITLE
Feature notify token purchase

### PR DESCRIPTION
**Pull request by MNS003**
Hey there,

Here's an awesome PR with some minor additions to how we build http messages.

Lemme know what you think.

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Prepaid Utility Service Interface Release Notes
```
Comment (TimElectrum) [inline release-notes.md:1]: release notes for our public APIs live here: https://github.com/electrumpayments/prepaidutility-service-interface/blob/master/src/docs/devguide/hugo/site/content/release-notes.md

Note the slightly different formatting compared to our internal release notes.

you can remove this page.

File: src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java [hunk 2]
```diff
@@ -69,7 +69,11 @@ public String toString() {
       FAULT_REPORT_REQUEST("FAULT_REPORT_REQUEST"),
       KEY_CHANGE_TOKEN_REQUEST("KEY_CHANGE_TOKEN_REQUEST"),
       CONFIRMATION_ADVICE("CONFIRMATION_ADVICE"),
-      REVERSAL_ADVICE("REVERSAL_ADVICE");
+      REVERSAL_ADVICE("REVERSAL_ADVICE"),
+      /**
+       * @since 3.10.1
```
Comment (TimElectrum) [inline src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java:74]: you sure that's the right version?

File: pom.xml [hunk 3]
```diff
@@ -44,8 +44,8 @@
     <swagger-maven-plugin-version>3.1.3</swagger-maven-plugin-version>
     <logback-version>1.0.13</logback-version>
     <major-version>3</major-version>
-    <minor-version>10</minor-version>
-    <patch-version>0</patch-version>
+    <minor-version>11</minor-version>
+    <patch-version>0-SNAPSHOT</patch-version>
```
Comment (TimElectrum) [inline pom.xml:48]: don't forget to drop `-SNAPSHOT`

**Review by TimElectrum — CHANGES_REQUESTED** [review]
please also update the `README.md` in the project's base directory

File: release-notes.md (hunk 1 above)
Comment (MNS003) [reply release-notes.md]: Awe, I didn't know that... Thanks.

File: src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java (hunk 2 above)
Comment (MNS003) [reply src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java]: I am now

File: src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java [hunk 4]
```diff
@@ -69,7 +69,11 @@ public String toString() {
       FAULT_REPORT_REQUEST("FAULT_REPORT_REQUEST"),
       KEY_CHANGE_TOKEN_REQUEST("KEY_CHANGE_TOKEN_REQUEST"),
       CONFIRMATION_ADVICE("CONFIRMATION_ADVICE"),
-      REVERSAL_ADVICE("REVERSAL_ADVICE");
+      REVERSAL_ADVICE("REVERSAL_ADVICE"),
+      /**
+       * @since 3.11.0
+       */
+      NOTIFY_TOKEN_PURCHASE("NOTIFY_TOKEN_PURCHASE");
```
Comment (TimElectrum) [inline src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java:76]: the `ModelTests` class in the test classes has a test to ensure the ordinal values of the `TokenTypeEnum` enumerations doesn't change. Please can you add a similar test for these `RequestType` enumerations.

**Review by TimElectrum — CHANGES_REQUESTED** [review]

File: src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java (hunk 4 above)
Comment (MNS003) [reply src/main/java/io/electrum/prepaidutility/model/ErrorDetail.java:76]: Awe, no problem.

**Review by TimElectrum — APPROVED** [review]

**Pull request merged**